great_expectations/expectations/expectation.py

```diff
@@ -2086,7 +2086,7 @@ def _validate(
             success = True
         elif nonnull_count > 0:
             success = _mostly_success(
-                nonnull_count, unexpected_count, self.get_success_kwargs().get("mostly")
+                nonnull_count, unexpected_count, self.get_success_kwargs().get("mostly", 1.0)
```

Comment:
@billdirks Is there any way we can follow the pattern of defaults in the configuration, rather than at the `Python` API level?  For example, perhaps now that all types of the `Expectation` accept the `mostly` parameter, we can add it to
```
    default_kwarg_values = {
        "include_config": True,
        "catch_exceptions": False,
        "result_format": "BASIC",
    }
```
at the class level.

Happy to discuss.  Thanks!

Reply:
Thanks for the discussion! Done.